pom.xml

```diff
@@ -335,7 +566,7 @@
                         <sourceDirectory>src/main/java</sourceDirectory>
                     </sourceDirectories>
                     <configLocation>${maven.multiModuleProjectDirectory}/config/checkstyle/checkstyle.xml</configLocation>
-                    <propertyExpansion>config_loc=${maven.multiModuleProjectDirectory}/config/checkstyle</propertyExpansion>
+                    <propertyExpansion>config_loc=${maven.multiModuleProjectDirectory}/config/checkstyle</propertyExpansion> <!-- Path needs to be synchronized with the caching config for checkstyle in the Develocity Caching Configuration (gradleEnterprise). -->
```

Comment:
Can you please create a Maven property for this and the other case, and reference it in both places?

Reply:
Updated 👍